Add info section about client-side Klaviyo events (#4087)
segmentio/segment-docs: src/connections/destinations/catalog/klaviyo/index.md

When you call `track` on `analytics.js`, Segment calls Klaviyo's `track` with the same parameters.

> info ""

> > When you're tracking client-side, some Klaviyo events require that an `identify` event **must** be sent first before a `track` event. 

Reviewer: ```suggestion
> When you're tracking client-side, some Klaviyo events require you send an Identify call  before a Track call. 
```
